src/pages/iou/request/step/IOURequestStepDistance.js

```diff
@@ -226,6 +239,7 @@ function IOURequestStepDistance({
                         onPress={submitWaypoints}
                         text={translate('common.next')}
                         isLoading={!isOffline && (isLoadingRoute || shouldFetchRoute || isLoading)}
+                        isDisabled={nonEmptyWaypointsCount<2}
```

Comment:
Why do you add this change?

Reply:
To avoid displaying err message when editing the waypoints first, I added it. 
As you know, in the previous commit, when user set the first waypoint, the err msg("please enter at least two waypoints") appears.

Reply:
@DylanDylann When should the err msg (atLeastTwoDifferentWaypointsError) be displayed?